tests/testthat/test-argufy_package.R

```diff
@@ -0,0 +1,32 @@
+{
+  temp_lib <- file.path(tempdir(), "lib")
+
+  old <- .libPaths()
+  .libPaths(temp_lib)
+  on.exit(.libPaths(old))
+  tools:::.install_packages("TestS4")
```

Comment:
CRAN doesn't check code in tests, but I agree it is not better than `install.packages`, I forgot you could pass file paths to `install.packages`.